src/components/ui/agreement-form.tsx

```
@@ -41,86 +48,92 @@ const AgreementForm = ({
     return (
-      <Link
-        href={link}
-        target={isExternal ? '_blank' : '_self'}
-        rel={isExternal ? 'noopener noreferrer' : undefined}
+      <button
+        className='px-5'
+        onClick={e => {
+          e.preventDefault();
+          setOpenTerm(termsKey);
+        }}
       >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)